node/src/chain_specs/manta.rs

```diff
@@ -198,4 +222,198 @@ pub fn manta_testnet_config() -> Result<MantaChainSpec, String> {
+                INITIAL_COLLATOR_BALANCE,
+            ), // COLLATOR 4
+            (
+                hex!("4039a894994d326e32801a0fc19a267383b6231e74bc9c3fd1021d949d849406").into(),
+                INITIAL_COLLATOR_BALANCE,
+            ), // COLLATOR 5
+            (
+                root_key.clone(),
+                TOTAL_ISSUANCE - 4 * INITIAL_CONTROLLER_BALANCE - 4 * INITIAL_COLLATOR_BALANCE,
+            ), // SUDO ACCOUNT
+        ];
+        const _: () = assert!(
+            NORMAL_COLLATOR_MINIMUM_STAKE < INITIAL_COLLATOR_BALANCE,
+            "won't be able to register collator, balance in account set too low"
+        );
+
+        GenesisConfig {
+            system: manta_runtime::SystemConfig {
+                code: manta_runtime::WASM_BINARY
+                    .expect("WASM binary was not build, please build it!")
+                    .to_vec(),
+            },
+            balances: manta_runtime::BalancesConfig {
+                balances: endowments,
+            },
+            // empty aura authorities, collators registered with parachain staking instead
+            aura: Default::default(),
+            sudo: manta_runtime::SudoConfig {
+                key: Some(root_key),
+            },
+            parachain_staking: ParachainStakingConfig {
+                candidates: genesis_collators
+                    .iter()
+                    .map(|collator| (collator.acc.clone(), NORMAL_COLLATOR_MINIMUM_STAKE))
+                    .collect(),
+                delegations: vec![],
+                // set to 0 inflation at genesis
+                inflation_config: manta_runtime::staking::inflation_config::<manta_runtime::Runtime>(
+                ),
+            },
+            parachain_info: manta_runtime::ParachainInfoConfig {
+                parachain_id: MANTA_PARACHAIN_ID.into(),
+            },
+            collator_selection: manta_runtime::CollatorSelectionConfig {
+                invulnerables: vec![],
+                candidacy_bond: 0,
+                ..Default::default()
+            },
+            session: manta_runtime::SessionConfig {
+                keys: genesis_collators
+                    .iter()
+                    .map(|collator| {
+                        (
+                            collator.acc.clone(),  // account id
+                            collator.acc.clone(),  // validator id
+                            collator.keys.clone(), // collator session keys
+                        )
+                    })
+                    .collect(),
+            },
+            parachain_system: Default::default(),
+            polkadot_xcm: calamari_runtime::PolkadotXcmConfig {
+                safe_xcm_version: Some(SAFE_XCM_VERSION),
+            },
+        }
+    }
+
+    pub fn manta_mainnet_config() -> MantaChainSpec {
+        let genesis_collators: Vec<Collator> = vec![
+        Collator::new( // c1: dfWh1oFWNxrmrHdiGdVpQQD6Pp3SmY5NrNsAGNRCxgaqJuZif
```

Comment:
@grenade pls doublecheck that collator addresses and session keys match with what you gave me

Reply:
all verified (c1 - c5, node, collator, nimbus, vrf).

Reply:
all verified (c1 - c5, node, collator, nimbus, vrf).